buildah.go

```diff
@@ -344,6 +344,8 @@ type BuilderOptions struct {
 	ProcessLabel string
 	// MountLabel is the SELinux mount label associated with the container
 	MountLabel string
+	// OCIHooksDir is the location of OCI hooks for the build containers
+	OCIHooksDir []string
```

Comment:
This duplicates `CommonBuildOpts.OCIHooksDir`.

Reply:
yes this is not needed.